force-app/main/default/classes/RD2_OpportunityEvaluation_BATCH.cls

```diff
@@ -51,6 +51,20 @@ public without sharing class RD2_OpportunityEvaluation_BATCH
         private set;
     }
 
+    /**
+    * @description Returns true if the "Exclude Closed Recurring Donations" setting is enabled
+    * Can be manually overwritten for use in Bulk Data Processes
+    */
+    public Boolean closedFilterEnabled {
+        get {
+            if (closedFilterEnabled == null) {
+                closedFilterEnabled = UTIL_CustomSettingsFacade.getRecurringDonationsSettings().Exclude_Closed_Recurring_Donations__c;
```

Comment:
If I'm not mistaken, Custom Settings checkbox fields can be null. Maybe this could do a null check and return false if null?